set-versions

```diff
@@ -38,7 +38,7 @@ for project in $(find . -name project.clj -exec dirname '{}' \;); do
 
   cljproj=$(basename $project)
 
-  if [ "$cljproj" == "Donkey" ]; then
+  if [ "$cljproj" == "Terrain" ]; then
```

Comment:
Are we no longer going to set versions in the project.clj files? I know we've been at `5.0.0` for a few releases now... Currently the `/status` endpoints use this version in their responses. Is that also changing when we use Docker labels for versions?